confectory-core/src/test/java/net/obvj/confectory/ConfectoryTest.java

```diff
@@ -0,0 +1,50 @@
+package net.obvj.confectory;
+
+import static net.obvj.junit.utils.matchers.AdvancedMatchers.instantiationNotAllowed;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+import org.junit.Test;
+
+import net.obvj.confectory.settings.ConfectorySettings;
+
+/**
+ * Unit tests for the {@link Confectory} class.
+ * 
+ * @author FernandoNSC (Fernando Tiannamen)
+ */
+public class ConfectoryTest
+{
+
+    ConfigurationContainer configurationContainer = new ConfigurationContainer();
```

Comment:
Because we have plans to modify the `equals()`) method of `ConfigurationContainer`, it's better to use a mock instead of an actual instance.